chainer_chemistry/functions/mean_squared_error.py

```diff
@@ -0,0 +1,77 @@
+import numpy
+
+from chainer import cuda
+from chainer import function_node
+import chainer.functions
+from chainer.utils import type_check
+
+
+class MeanSquaredError(function_node.FunctionNode):
+
+    """Mean squared error (a.k.a. Euclidean loss) function."""
+
+    def __init__(self, ignore_nan=False):
```

Comment:
IMO, what "ignore" means can differ from people to people. So, I would suggest more descriptive option name "e.g. convert_nan_to_zero" or write what this option does in detail in the document.